integrations/tensorflow/e2e/keras/train/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set(REFERENCE_BACKEND tf)
+
+function(add_e2e_test filename target_backend labels)
+  iree_package_ns(_PACKAGE_NS)
+  string(REPLACE "::" "/" _PACKAGE_PATH ${_PACKAGE_NS})
+  string(REPLACE ".py" "" _name ${filename})
+
+  set(_name "${_PACKAGE_PATH}/${_name}__${target_backend}")
```

Comment:
I was curious about this naming, so I looked at the [presubmit logs](https://source.cloud.google.com/results/invocations/a6072489-54a0-423c-9a5c-b12e886c18d9/targets/iree%2Fgcp_ubuntu%2Fcmake-bazel%2Flinux%2Fx86-swiftshader%2Fpresubmit/log) to see the full list of targets. I don't see these new tests there though? And the existing tests have names like `tensorflow_e2e__resource_ops_test.py__iree_vmla` with `.py` included in the target names, unlike here?

Reply:
There is no unity in how were creating these targets right now. I like being able to select tests by using the directory structure (that's how they were laid out for Bazel). `tensorflow_e2e` is from the rule Stella created in `integrations/tensorflow/e2e/CMakeLists.txt`. The idea here was to roughly match `iree_py_test`'s convention.

The targets weren't showing up because the test invocation was using `tensorflow_e2e`. Should be updated correctly now!

Reply:
Yep, test names matching directory structure is helpful :D